pkgs/applications/science/electronics/horizon-eda/default.nix

```diff
@@ -1,75 +1,33 @@
 { stdenv
 , boost
+, callPackage
 , coreutils
-, cppzmq
-, curl
-, libepoxy
-, fetchFromGitHub
-, glm
-, gtkmm3
-, lib
-, libarchive
-, libgit2
-, librsvg
 , libspnav
-, libuuid
-, opencascade-occt
-, pkg-config
-, podofo
 , python3
-, sqlite
 , wrapGAppsHook
 }:
 
+let
+  base = callPackage ./base.nix { };
+in
 stdenv.mkDerivation rec {
-  pname = "horizon-eda";
-  version = "2.5.0";
+  inherit (base) pname version src meta CASROOT;
```

Comment:
With this formulation you're not adding yourself as a maintainer. Unless @guserav want to take upon himself the maintainership I'd assume you intended to add yourself?

Reply:
I've added myself to the maintainers list